rs/sns/init/src/gen/ic_sns_init.pb.v1.rs

```diff
@@ -275,7 +274,7 @@ pub struct FractionalDeveloperVotingPower {
     /// The swap bucket.
     #[prost(message, optional, tag = "3")]
     pub swap_distribution: ::core::option::Option<SwapDistribution>,
-    /// The airdrop bucket.
+    /// OBSOLETE.
```

Comment:
So it's an option - what's needed to fully delete it?

Reply:
(NCR, just wondering if there's a plan where we eventually drop the field entirely)

Reply:
Good point, let me remove it right after this PR.